components/SubmissionForm.vue

```diff
@@ -1,163 +1,185 @@
+            >
+                {{ $t('submitPage.heading') }}
+            </h1>
+            <p
+                data-testid="submit-subheading"
+                class="mb-10 w-96 text-center text-primary-text-muted text-sm font-normal font-sans"
             >
-                {{ $t('submitPage.lastNameValidation') }}
-            </span>
+                {{ $t('submitPage.subheading') }}
+            </p>
             <span
-                :class="[
-                    !isValidInput.firstName.value ? 'visible' : 'invisible',
-                    'w-44',
-                ]"
+                class="mb-2 text-primary-text text-sm font-normal font-sans"
+            >{{ $t('submitPage.googleMaps') }}</span>
+            <p
+                v-show="!isValidInput.googleMapsUrl.value"
+                class="text-error text-xs font-sans"
             >
-                {{ $t('submitPage.firstNameValidation') }}
-            </span>
-        </p>
-        <div>
+                {{ $t('submitPage.googleMapsValidation') }}
+            </p>
             <input
-                v-model="submissionStore.lastName"
-                data-testid="submit-input-lastname"
-                class="mb-5 mr-1 px-3 py-3.5 w-48 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
-            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                v-model="location"
+                data-testid="submit-input-googlemaps"
                 type="text"
                 required
-                maxlength="30"
-                :placeholder="$t('submitPage.lastName')"
-                @blur="initialValidationCheck(submissionStore.lastName, 'lastName')"
+                class="mb-5 px-3 py-3.5 w-96 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
+                text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                :placeholder="$t('submitPage.location')"
+                @blur="initialValidationCheck(location, 'googleMaps')"
             >
-            <input
-                v-model="submissionStore.firstName"
-                data-testid="submit-input-firstname"
-                class="mb-5 px-3 py-3.5 w-48 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
-            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
-                type="text"
-                maxlength="30"
-                :placeholder="$t('submitPage.firstName')"
-                @blur="initialValidationCheck(submissionStore.firstName, 'firstName')"
+            <span
+                class="mb-2 text-primary-text text-sm font-normal font-sans"
+            >{{ $t('submitPage.healthcareProfessionalName') }}</span>
+            <p class="flex text-error text-xs font-sans">
+                <span
+                    :class="[
+                        !isValidInput.lastName.value ? 'visible' : 'invisible',
```

Comment:
I think you can just use v-show or v-if here.
If the page layout messes up, you can have this in a container with a fixed size that doesn't disappear and then just conditionally render the element within.